components/motor/gpiostepper/gpiostepper.go

```diff
@@ -108,7 +112,7 @@ func newGPIOStepper(ctx context.Context, b board.Board, mc AttrConfig, name stri
 	m := &gpioStepper{
 		theBoard:         b,
 		stepsPerRotation: mc.TicksPerRotation,
-		stepperDelay:     mc.StepperDelay,
+		stepperDelay:     time.Duration(mc.StepperDelay) * time.Microsecond, // TODO (rh) maybe unnecessary
```

Comment:
nit: you don't need `time.Duration`. you can just do `mc.StepperDelay * time.Microsecond`

Reply:
no because this is an integer coming in from the config